kinesumer.go

```diff
@@ -229,6 +253,16 @@ func NewKinesumer(cfg *Config) (*Kinesumer, error) {
 	if kinesumer.efoMode {
 		kinesumer.efoMeta = make(map[string]*efoMeta)
 	}
+	if cfg.CheckPointInterval > 0 {
+		kinesumer.checkPointInterval = cfg.CheckPointInterval
+	}
+	if cfg.CheckPointTimeout > 0 {
+		kinesumer.checkPointTimeout = cfg.CheckPointTimeout
+	}
+
+	if !cfg.AutoCommitEnabled {
```

Comment:
Default of `AutoCommitEnabled` is false. If a client don't pass true, auto commit mode will be disabled. But the auto commit mode should be enabled as default. So we need to provide a function returning a config has default values.